tests/mysql/tox.ini

```diff
@@ -29,8 +29,9 @@ passenv = *
 commands =
     docker-compose stop mysqldb
     bash -ec "if [ `uname` = 'Linux' ] && [ "{env:GITHUB_ACTIONS:false}" != "true" ]; then chcon -h system_u:object_r:bin_t:s0 sql; chcon -Rt svirt_sandbox_file_t sql; fi"
+    bash -ec "if [ `uname` = 'Linux' ] && [ "{env:GITHUB_ACTIONS:false}" != "true" ]; then chcon -h system_u:object_r:bin_t:s0 sql; chcon -Rt svirt_sandbox_file_t docker-healthcheck; fi"
```

Comment:
@rcbjBlueMars do you mind dropping a comment on where this line come from and what it does?

Reply:
When selinux is enabled, this chcon command can be used to grant permissions needed to mount the file/directory as a volume in a docker container.  The GITHUB_ACTIONS environment variable indicates whether tox is running locally or in a github action. This is only validon Linux.